src/Core/Utilities/EncryptedStringAttribute.cs

```diff
@@ -16,6 +16,7 @@ public class EncryptedStringAttribute : ValidationAttribute
         [EncryptionType.AesCbc256_B64] = 2, // iv|ct
         [EncryptionType.AesCbc128_HmacSha256_B64] = 3, // iv|ct|mac
         [EncryptionType.AesCbc256_HmacSha256_B64] = 3, // iv|ct|mac
+        [EncryptionType.XChaCha20Poly1305_B64] = 1, // cose bytes
```

Comment:
Can we add tests for this at `test/Core.Test/Utilities/EncryptedStringAttributeTests.cs`?

If we aren't sure what it will look like, yet I'm good with just having a follow-up ticket for it.